test/test_web.py

```diff
@@ -141,13 +141,65 @@ def test_main_menu(self) -> None:
         submenu_entry = submenu.find_element(By.XPATH, './/a[contains(@class, "item")]')
         self.assertFalse(submenu_entry.is_displayed())
         submenu.click()
+        actions = ActionChains(self.driver)
+        actions.move_to_element(submenu).perform()  # do a hoover
+
         self.assertIn("Bar", submenu_entry.text)
         self.assertTrue(submenu_entry.is_displayed())
         submenu_entry_href = submenu_entry.get_attribute('href')
         assert submenu_entry_href is not None
         self.assertEqual(submenu_entry_href.strip(), "http://localhost:42080/page/another_page/")
         submenu_entry.find_element(By.CSS_SELECTOR, 'i.bars.icon')
 
+    def test_main_menu_selection(self) -> None:
+        """Test that the clicked menu item is selected thus highlighted."""
+        self.server.page('index', menu_entry="Home", menu_icon='home')
+        self.server.page('overview', menu_entry="Foo", menu_icon='info')
+
+        # add submenu
```

Comment:
This comment is slightly inaccurate, as these method calls do not only add a submenu, but also new web pages.